cypress/integration/console/shared/header.spec.js

```diff
@@ -0,0 +1,73 @@
+// Copyright © 2020 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+describe('Header', () => {
+  describe('Console logout', () => {
+    const logout = userName => {
+      cy.findByTestId('profile-dropdown')
```

Comment:
More realistic selection would be something resembling

> find a dropdown containing your name in the header section

Reply:
Indeed. I think it also makes sense use `cy.within` when testing shared components. Added this to the docs.